contrib/templates/data-engineering/template/{{.project_name}}/scripts/test.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+#
+# test.py runs the unit tests for this project using pytest.
```

Comment:
Please explain when to use this versus just running pytest.

You explained in a comment above, can be copied here.